### ISSUE TITLE
Switch to .NET Core and license friendly dependencies for Eto.Wpf

### ISSUE THREAD

**Issue by cwensley** [picoe/Eto#1807]
Currently we have these nuget dependencies for Eto.Wpf, both do not support .NET Core and one is stuck at an old version due to licensing restrictions.

- [Windows7APICodePack-Shell](https://www.nuget.org/packages/Windows7APICodePack-Shell/)
- [Extended.Wpf.Toolkit](https://www.nuget.org/packages/Extended.Wpf.Toolkit/)

These should be switched to .NET Core friendly versions, however it may mean we need to bump up the minimum .NET framework for Eto.Wpf.dll. Alternatives that I can find are:

- [Microsoft-WindowsAPICodePack-Shell](https://www.nuget.org/packages/Microsoft-WindowsAPICodePack-Shell/) (requires .NET 4.5.2 vs 4.5)
- [DotNetProjects.Extended.Wpf.Toolkit](https://www.nuget.org/packages/DotNetProjects.Extended.Wpf.Toolkit/).

See #1544

**Comment by MartinRothschink** [comment]
I'm fine with 4.5.2 as minimum .Net version.

**Comment by Miepee** [comment]
With #2193 and the release of 2.7.0, is this issue fixed now?

**Comment by cwensley** [comment]
Nope, Eto.Wpf is still using Extended.Wpf.Toolkit, but to keep "oss license friendly" and generally free, you must use only the version that Eto references, which is 3.6.0.  Switching that out will be a huge breaking change for anyone that has made wpf-specific changes such as xaml styles, code styles, etc as the assembly name is different.  Not sure how to switch that out without breaking all that code just yet.

**Comment by dsoronda** [comment]
How about, create Eto.WpfCore project and move from there with new license (and cleaner code) ?

**Comment by philstopford** [comment]
Just want to link this to https://github.com/picoe/Eto/pull/2307

**Comment by cwensley** [comment]
Ok, this should now be done with #2735.  I've opted to remove the use of Extended.Wpf.Toolkit completely, as there were only a few controls that used it.
